app/models/service_load_balancer.rb

```diff
@@ -0,0 +1,68 @@
+class ServiceLoadBalancer < Service
+  include ServiceLoadBalancerMixin
+  include ServiceLoadBalancerOptionsMixin
```

Comment:
We can include `ServiceOrchestrationOptionMixin` and not to introduce the almost identical `ServiceLoadBalancerOptionsMixin`, just like what `ServiceAnsibleTower` does.